### ISSUE TITLE
Impossible to use Payment Service - getExistingPurchases => onFailure always crashes the app

### ISSUE THREAD

**Issue by Kra-Kra** [blackberry/BB10-WebWorks-Framework#673]
observed on: dev alpha b, OS 10.1.0.1485, WebWorks SDK 1.0.4.11, production environment (app is published in App World)

app calls 

blackberry.payment.getExistingPurchases(true, onSuccess, onFailure);

a couple of seconds after framework calls the onFailure callback, app becomes unresponsive and some time later it "disappears" (it is closed by the OS).

Sample code can be found in documentation: 

https://developer.blackberry.com/html5/api/blackberry.payment.html#.getExistingPurchases^2

*\* Update: *\* 
Same problem is observed for the following API call: 

blackberry.payment.checkExisting

In short - it is currently not possible to use Payment Service from within bb10 WebWorks app.
